### PR TITLE
fix(sample): boot failure on emulator

### PR DESCRIPTION
**Pull request by freeboub**
## Summary
This PR fixes 2 issues:
- remove unused from this comment: https://github.com/TheWidlarzGroup/react-native-video/pull/3867#pullrequestreview-2175363083
- Fix symbol issue at app boot:

07-17 14:05:36.352 23802 23802 F DEBUG   : Abort message: 'JNI DETECTED ERROR IN APPLICATION: JNI NewGlobalRef called with pending exception java.lang.NoSuchMethodError: no static or non-static method "Lcom/facebook/react/devsupport/CxxInspectorPackagerConnection$WebSocketDelegate;.didFailWithError(Ljava/util/OptionalInt;Ljava/lang/String;)V"

### Motivation
A working and clean sample app

## Test plan
Build and run the sample

### CODE REVIEW

**Review by moskalakamil — APPROVED** [review]

File: examples/basic/android/app/build.gradle [hunk 1]
```diff
@@ -83,8 +83,10 @@ android {
     namespace "com.videoplayer"
 
     compileOptions {
-        // These options are necessary to be able to build fro source
-        coreLibraryDesugaringEnabled true
+        // These options are necessary to be able to build from source
```
Comment (romansavka-mq) [inline examples/basic/android/app/build.gradle:86]: `coreLibraryDesugaring 'com.android.tools:desugar_jdk_libs:2.0.4'` in dependencies block is also related and should be commented out

**Review by KrzysztofMoch — APPROVED** [review]
apart from @romansavka-mq comment, LTGM

File: examples/basic/android/app/build.gradle (hunk 1 above)
Comment (freeboub) [reply examples/basic/android/app/build.gradle:86]: As it do not causes any issue, I think it is better to keep it here ...
I want users to do the smallest change possible to enable build from sources.
Is it ok ? @romansavka-mq @KrzysztofMoch ?

Comment (KrzysztofMoch) [reply examples/basic/android/app/build.gradle:86]: For me it's okay

Comment (romansavka-mq) [reply examples/basic/android/app/build.gradle:86]: It strange that doesn't cause issues, I thought that `coreLibraryDesugaring` configuration will not exist if `coreLibraryDesugaringEnabled` is not set to true. But if there is no problem than it's okay

**Review by romansavka-mq — APPROVED** [review]

**Pull request merged**